### PR TITLE
added prediction-retrodiction refs

### PR DESCRIPTION
**Pull request by xinmingxu**
### List of citation keys added (one per line)
Hawk85, 
Dess07, 
HassEtal20, 
PillEtal15,
RichSaxe20, 
MlodBrun14, 
BialEtal01, 
MahrCsib18, 
KostSaxe13, 
JonePash07, 
LangEtal15, 
DemiEtal18, 
Bord08, 
ThorTami20, 
Cove94, 
TamiThor18, 
ChanEtal20, 
CohnEtal20, 
EaglSejn00, 
HirsEcht12

### CODE REVIEW

File: cdl.bib [hunk 1]
```diff
@@ -1,3 +1,173 @@
...
+	Author = {Jorie Koster-Hale and Rebecca Saxe},
+	Journal = {Neuron},
+	Number = {5},
+	Pages = {836--848},
+	Title = {Theory of mind: A neural prediction problem},
+	Volume = {79},
+	Year = {2013}}
+
+@article{EaglSejn00,
+	Author = {David M Eagleman and Terrence J Sejnowski},
```
Comment (jeremymanning) [inline cdl.bib:64]: correct author formatting...

File: cdl.bib [hunk 2]
```diff
@@ -1,3 +1,173 @@
...
+	Journal = {Science},
+	Number = {5460},
+	Pages = {2036--2038},
+	Title = {Motion integration and postdiction in visual awareness},
+	Volume = {287},
+	Year = {2000}}
+
+@article{BialEtal01,
+	Author = {William Bialek and Ilya Nemenman and Naftali Tishby},
```
Comment (jeremymanning) [inline cdl.bib:73]: author formatting...

File: cdl.bib [hunk 3]
```diff
@@ -1,3 +1,173 @@
...
+	Journal = {Neural Computation},
+	Number = {11},
+	Pages = {2409--2463},
+	Title = {Predictability, complexity, and learning},
+	Volume = {13},
+	Year = {2001}}
+
+@article{JonePash07,
+	Author = {Jason Jones and Harold Pashler},
```
Comment (jeremymanning) [inline cdl.bib:82]: author formatting...

File: cdl.bib [hunk 4]
```diff
@@ -1,3 +1,173 @@
...
+	Journal = {Psychonomic Bulletin and Review},
+	Number = {2},
+	Pages = {295--300},
+	Title = {Is the mind inherently forward looking? Comparing prediction and retrodiction},
+	Volume = {14},
+	Year = {2007}}
+
+@article{PillEtal15,
+	Author = {David B Pillemer and Kristina L Steiner and Kie J Kuwabara and Dorthe Kirkegaard Thomsen and Connie Svob},
```
Comment (jeremymanning) [inline cdl.bib:91]: formatting

File: cdl.bib [hunk 5]
```diff
@@ -1,3 +1,173 @@
...
+	Journal = {Consciousness and Cognition},
+	Pages = {233--245},
+	Title = {Vicarious memories},
+	Volume = {36},
+	Year = {2015}}
+
+@article{MlodBrun14,
+	Author = {Leonard Mlodinow and Todd A Brun},
```
Comment (jeremymanning) [inline cdl.bib:99]: formatting

File: cdl.bib [hunk 6]
```diff
@@ -1,3 +1,173 @@
...
+	Journal = {Physical Review {E}},
+	Number = {5},
+	Pages = {052102},
+	Title = {Relation between the psychological and thermodynamic arrows of time},
+	Volume = {89},
+	Year = {2014}}
+
+@article{Dess07,
+	Author = {Jean-Louis Dessalles},
```
Comment (jeremymanning) [inline cdl.bib:108]: formatting

File: cdl.bib [hunk 7]
```diff
@@ -1,3 +1,173 @@
...
+	Journal = {Behavioral and Brain Sciences},
+	Number = {3},
+	Pages = {321--322},
+	Publisher = {Cambridge {University} Press},
+	Title = {Storing events to retell them},
+	Volume = {30},
+	Year = {2007}}
+
+@article{MahrCsib18,
+	Author = {Johannes B Mahr and Gergely Csibra},
```
Comment (jeremymanning) [inline cdl.bib:118]: formatting

**Review by jeremymanning — CHANGES_REQUESTED** [review]
for all entries:
- check/fix formatting of author and editor names ([instructions](https://github.com/ContextLab/CDL-bibliography#verify))

**Comment by jeremymanning** [comment]
@xinmingxu i'm going to close this request for now (until the conflicts are resolved)

**Comment by xinmingxu** [comment]
How to correct author formatting automatically? Seems not supported by 'autofix' option.

**Comment by jeremymanning** [comment]
Have you read the readme file to make sure you've formatted things correctly?  If there's a case the format doesn't handle, you can add a "force" field to the entry and the checker will skip it

**Comment by xinmingxu** [comment]
I can always correct author formatting manually, but an automatic way to do this will be great (currently the checker doesn't pick up full first name formatting)

**Pull request closed without merging**